evalai/get_starters.py

```diff
@@ -0,0 +1,13 @@
+import click
+import subprocess
+
+from evalai.utils.config import STARTERS_URL
+
+
+@click.group(invoke_without_command=True)
+def get_starters():
```

Comment:
Can we please change this to `download_evalai_starter_kit`

Reply:
Sure! I was also thinking that there would be a better name than `get_starters`, but could not find it due to my intermediate English level...